DuckDuckGo/WindowsBrowserWaitlistView.swift

```diff
@@ -0,0 +1,369 @@
+                    AllowNotificationsView(action: action)
+                        .padding(.top, 4)
+                } else {
+                    Button(UserText.waitlistNotifyMe) {
+                        action(.requestNotificationPermission)
+                    }
+                    .buttonStyle(RoundedButtonStyle(enabled: true))
+                    .padding(.top, 32)
+                }
+            }
+
+            Spacer()
+        }
+        .padding([.leading, .trailing], 24)
+        .multilineTextAlignment(.center)
+    }
+
+}
+
+private struct AllowNotificationsView: View {
+
+    let action: (WaitlistViewModel.ViewAction) -> Void
+
+    var body: some View {
+
+        VStack(spacing: 20) {
+
+            Text(UserText.waitlistNotificationDisabled)
+                .font(.proximaNova(size: 17))
+                .foregroundColor(.waitlistText)
+                .fixMultilineScrollableText()
+                .lineSpacing(5)
+
+            Button(UserText.waitlistAllowNotifications) {
+                action(.openNotificationSettings)
+            }
+            .buttonStyle(RoundedButtonStyle(enabled: true))
+
+        }
+        .padding(24)
+        .background(Color.waitlistNotificationBackground)
+        .cornerRadius(8)
+        .shadow(color: .black.opacity(0.05), radius: 12, x: 0, y: 4)
+
+    }
+
+}
+
+// MARK: - Invite Available Views
+
+private struct ShareButtonFramePreferenceKey: PreferenceKey {
+    static var defaultValue: CGRect = .zero
+    static func reduce(value: inout CGRect, nextValue: () -> CGRect) {}
+}
+
+struct WindowsBrowserWaitlistInvitedView: View {
+
+    let inviteCode: String
+    let action: (WaitlistViewModel.ViewAction) -> Void
+
+    @State private var shareButtonFrame: CGRect = .zero
+
+    var body: some View {
+        GeometryReader { proxy in
+            ScrollView {
+                VStack(alignment: .center, spacing: 0) {
+                    HeaderView(imageName: "WaitlistInvited", title: UserText.waitlistYoureInvited)
+
+                    Text(UserText.windowsWaitlistInviteScreenSubtitle)
+                        .font(.proximaNova(size: 17))
+                        .foregroundColor(.waitlistText)
+                        .padding(.top, 16)
+                        .lineSpacing(6)
+                        .fixedSize(horizontal: false, vertical: true)
+
+                    Text(UserText.waitlistInviteScreenStepTitle(step: 1))
+                        .font(.proximaNova(size: 17, weight: .bold))
+                        .foregroundColor(.waitlistText)
+                        .padding(.top, 28)
+                        .padding(.bottom, 8)
+
+                    Text(UserText.windowsWaitlistInviteScreenStep1Description)
+                        .font(.proximaNova(size: 17))
+                        .foregroundColor(.waitlistText)
+                        .lineSpacing(6)
+
+                    if #available(iOS 14.0, *) {
```

Comment:
We now have iOS 14 as our minimum deployment target, so the `else` branch here can go. 🎉